fluentd/lib/fluent-plugin-prometheus/in_prometheus_tail_monitor.rb

```diff
@@ -0,0 +1,108 @@
+require 'fluent/plugin/input'
+require 'fluent/plugin/in_monitor_agent'
+require 'fluent/plugin/prometheus'
+
+module Fluent::Plugin
+  class PrometheusTailMonitorInput < Fluent::Plugin::Input
+    Fluent::Plugin.register_input('prometheus_tail_monitor', self)
```

Comment:
If we use this plugin, change the name to avoid confusion with the fluentd built-in.
Can we just add this logic to the other plugin, so we only have one? 
We're always going to want the metric functionality so I don't see much benefit in splitting them

Reply:
need further discussions

Reply:
class PrometheusTailMontiorInput is renamed to  PrometheusTailMontiorInputMetric